packages/ui/package.json

```diff
@@ -38,6 +38,7 @@
   "dependencies": {
     "@faststore/components": "*",
     "@reach/popover": "^0.16.0",
+    "@storybook/preset-scss": "^1.0.3",
```

Comment:
storybook is only used on dev mode, so can you move this dependency to `devDependencies`?

Reply:
another thing, since we use preset-scss by storybook, can we remove `css-loader` `sass-loader` and `style-loader`?

Reply:
> another thing, since we use preset-scss by storybook, can we remove `css-loader` `sass-loader` and `style-loader`?

no.. we still [need](https://www.npmjs.com/package/@storybook/preset-scss) them.